### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by mindys360**

### CODE REVIEW

File: student-work/2022/Mindy-Steen/ReadMe.md [hunk 1]
```diff
@@ -4,11 +4,11 @@
 Self-directed, collaborative, and energetic Technical Communicator with solid understanding of the way enterprise, B2B SaaS businesses operate. Self disciplined focused, results-driven individual who stays current with industry changes through networking and ongoing professional development. Highly motivated to contribute to your documentation team and to create content that customers love. 
 
 ## Contact ##
-[LinkedIn](https://www.linkedin.com/in/mindysteen360/) | [Email](mindys360@gmail.com) | [Phone](053-232-0454)
+[LinkedIn](https://www.linkedin.com/in/mindysteen360/) | [Email](mindys360@gmail.com)
```
Comment (Laura-Novich-OBW) [inline student-work/2022/Mindy-Steen/ReadMe.md:7]:
```suggestion
[LinkedIn](https://www.linkedin.com/in/mindysteen360/) | [Email](mailto:mindys360@gmail.com) 
```

**Review by Laura-Novich-OBW — COMMENTED** [review]
small improvement - adding mailto: to your email will open your default email app and populate the TO field with your email address.

**Comment by Laura-Novich-OBW** [comment]
Please look at the small change
and approve or reject

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
please look at the comment and either accept or reject the changes
looking good!

**Comment by lsnovich** [comment]
Seems to me that this change was already done in the PR I just merged. 
https://github.com/Laura-Novich-OBW/student-showcase/pull/101
If this is not the case let me know and I'll re-open this one. \
Otherwise, if the PR#101 is correct let this one stay closed

**Pull request closed without merging**